tests/dune

```diff
@@ -1,4 +1,4 @@
 (tests
  (names utils ok_ko)
- (deps (glob_files OK/*.lp) (glob_files KO/*.lp) lambdapi.pkg)
+ (deps (glob_files OK/*.lp) (glob_files "OK/a b/*.lp") (glob_files KO/*.lp) lambdapi.pkg)
  (libraries lambdapi.core lambdapi.handle lambdapi.tool alcotest))
```

Comment:
Alt-ergo <= 2.3.3 is needed for running tests. Shouldn't this dependency appear somewhere?

Reply:
It's more a binary than a caml library isn't it?

Reply:
What do you mean?

Reply:
It only makes sense to appear as a dependency if Dune may know how to build it [for example by composition]